### ISSUE TITLE
Need a CSS class for login/register message

### ISSUE THREAD

**Issue by q2apro** [q2a/question2answer#484]
![extra class for login message](https://cloud.githubusercontent.com/assets/6764548/21589477/d98cd7ea-d0fa-11e6-9457-f3179a817a4b.png)

No chance to assign a special CSS style to it. Because it has no extra class for the error / warning...

**Comment by q2apro** [comment]
public function a_form($a_form)
	{
		var_dump($a_form);

gives me: 

array(4) { ["title"]=> string(254) "Please login or register..." ["id"]=> string(4) "anew" ["collapse"]=> bool(true) ["style"]=> string(4) "tall" }

**Comment by q2apro** [comment]
Idea for a fix, checking for "typical" characteristics (but it's more a hack...):

	public function a_form($a_form)
	{
		$csswarning = '';
		if( empty($a_form['tags']) && empty($a_form['fields']) && isset($a_form['id']) )
		{
			if($a_form['id']=='anew')
			{
				$csswarning = ' qa-error';
			}
		}
		
		$this->output('<div class="qa-a-form'.$csswarning.'"'.(isset($a_form['id']) ? (' id="'.$a_form['id'].'"') : '').
			(@$a_form['collapse'] ? ' style="display:none;"' : '').'>');

		$this->form($a_form);
		$this->c_list(@$a_form['c_list'], 'qa-a-item');

		$this->output('</div> ', '');
	}

**Comment by q2apro** [comment]
**Same for the comment message**

Temporary fix: 

public function c_form($c_form)
		{
			$csswarning = '';
			if( empty($c_form['tags']) && empty($c_form['fields']) && isset($c_form['collapse']) )
			{
				if($c_form['collapse'])
				{
					$csswarning = ' qa-error';
				}
			}
			
			$this->output('<div class="qa-c-form'.$csswarning.'"'.(isset($c_form['id']) ? (' id="'.$c_form['id'].'"') : '').
				(@$c_form['collapse'] ? ' style="display:none;"' : '').'>');

			$this->form($c_form);

			$this->output('</div> ', '');
		}

**Comment by svivian** [comment]
> Idea for a fix, checking for "typical" characteristics (but it's more a hack...)

Yes that probably isn't the best way to do things. It would make more sense to add the message as an error/warning at the question-view.php line you showed in the first comment. The "please log in" doesn't need to be a title.

There is also a "qa-warning" class we could use, but I don't recall off-hand whether there's a way to add warning to forms easily.
